assets/xml/objects/object_d_hsblock.xml

```diff
@@ -1,10 +1,10 @@
 ﻿<Root>
     <File Name="object_d_hsblock" Segment="6">
```

Comment:
For fully-documented objects like this, we like to have a comment above the \<File\> node describing what they are like so:
```suggestion
    <!-- Assets for the hookshot post -->
    <File Name="object_d_hsblock" Segment="6">
```